scripts/check.sh

```diff
@@ -1,10 +1,8 @@
 #!/bin/bash
 
+# Run flake8 against the Django codebase
+
 set -e
 set -x
 
-# Run flake8 against the Django codebase and output a known string so that
```

Comment:
I think that that comment is only important in `test.sh`, where it was copied from. I think `check.sh` is just being used by developers for a quick lint.